target/linux/ipq807x/image/generic.mk

```diff
@@ -173,3 +173,20 @@ define Device/zyxel_nbg7815
 		kmod-bluetooth
 endef
 TARGET_DEVICES += zyxel_nbg7815
+
+define Device/yuncore_ax880
+	$(call Device/FitImage)
+	$(call Device/UbiFit)
+	DEVICE_VENDOR := Yuncore
+	DEVICE_MODEL := AX880
+	BLOCKSIZE := 128k
+	PAGESIZE := 2048
+	DEVICE_DTS_CONFIG := config@hk09
+	SOC := ipq8072
+	DEVICE_PACKAGES := ipq-wifi-yuncore_ax880
+	IMAGES := sysupgrade.bin factory.bin factory.ubi	
```

Comment:
`IMAGES += factory.bin` doesn't work?

Reply:
factory.bin is not enough. U-boot allow to flash factory.bin but it is a lot more complicated. 

To flash ubi file there only 2 commands needed:
```
tftpboot server_ip:file.ubi
flash rootfs
```
So i prefer factory.ubi

Reply:
You are not getting my point, `Device/UbiFit` already defines `IMAGES := factory.ubi sysupgrade.bin`.
So my question was whether simply extending the variable via `IMAGES += factory.bin` works instead of redefining it?

Reply:
Also, where is the recipe for `factory.bin`, cause I dont see one at all?

Reply:
> You are not getting my point, `Device/UbiFit` already defines `IMAGES := factory.ubi sysupgrade.bin`. So my question was whether simply extending the variable via `IMAGES += factory.bin` works instead of redefining it?

Now i got it. Append factory.bin